src/main/java/net/neoforged/neoforge/common/conditions/IConditionBuilder.java

```diff
@@ -17,12 +19,12 @@ default ICondition and(ICondition... values) {
         return new AndCondition(List.of(values));
     }
```

Comment:
How about we make all these methods static and turn this class into a helper class? E.g. `NeoForgeConditions.never()`. That would match what we do in other places better.